_data/projects/MindsDB.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: MindsDB
+desc: An Explainable AutoML framework for developers built on top of Pytorch. I
+site: https://github.com/mindsdb/mindsdb
+tags:
```

Comment:
Can you please add more tags here like `machine learning, pytorch, ludwig, tensorflow, artificial intelligence`